G3GridView.podspec

```diff
@@ -8,7 +8,8 @@ Pod::Spec.new do |s|
   s.swift_version = '5.0'
   s.ios.deployment_target = '9.0'
   s.source       = { :git => "https://github.com/KyoheiG3/GridView.git", :tag => s.version.to_s }
-  s.source_files  = "GridView/**/*.{h,swift}"
+  s.source_files = "GridView/**/*.{h,m,swift}"
+  s.module_name = "GridView"
```

Comment:
Required for the import statement to work

https://github.com/KyoheiG3/GridView/blob/20dfa888af11f79a622e2b647a45b329fe7d7b48/GridView/GridView%2BDelegateProxy.m#L10